### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by Dahsle** [GetPublii/Publii#1514] [Help] [answered]
Please tell me how to add a favicon, I can’t do it.

**Comment by Dahsle** [comment]
Installed solved

**Comment by candideu** [comment, marked as answer]
Theme > Additional > Upload favicon file

![image](https://github.com/GetPublii/Publii/assets/55474996/9cba674c-df39-42ec-a1b9-5ac21120e847)

**Reply by Dahsle** [reply]
Thank you very much I installed it.
